### PR TITLE
Ebra branch

### PR DESCRIPTION
**Pull request by ebrahim02-ag**
Bar Graph

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -35,3 +35,4 @@ yarn-error.log*
 .vercel
 package-lock.json
 .vscode
+.env.sample
```
Comment (NirmalHegde) [inline .gitignore:38]: remove this, it didnt rly do much when we added it loool

File: components/charts/bar.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        setData(data);
+      },
+      (err) => {
+        console.log("Error fetching firebase snapshot! " + err);
+      }
+    );
+
+  return (
+    <div>
+      {/* <div className="header">
```
Comment (NirmalHegde) [inline components/charts/bar.js:38]: is this comment necessary or can we remove?

File: components/charts/bar.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        <h1 className="title">Doughnut</h1>
+      </div> */}
+      <div className="chart">
+        <Bar
+          data={{
+            labels: data.label,
+            datasets: [
+              {
+                label: "# of Students",
```
Comment (NirmalHegde) [inline components/charts/bar.js:47]: is this supposed to be a generic component? if it is, this should be based on data in firebase

**Review by NirmalHegde — COMMENTED** [review]
See comments.

File: components/charts/bar.js (hunk 3 above)
Comment (NirmalHegde) [reply components/charts/bar.js:47]: label***

Comment (NirmalHegde) [reply components/charts/bar.js:47]: I dont think it will always be # of students rii so

**Review by parminpatel28 — APPROVED** [review]

**Review by eevanwong — APPROVED** [review]

**Comment by eevanwong** [comment]
I forgot to push the changes so there are no segments in demographics pg, we'll have to resolve that later.

**Comment by parminpatel28** [comment]
> I forgot to push the changes so there are no segments in demographics pg, we'll have to resolve that later.

you can just pull latest from develop when this is merged and bring in your changes

**Pull request merged**